torch_sparse/narrow.py

```diff
@@ -1,11 +1,69 @@
 from typing import Tuple
 
+import torch
 from torch_sparse.storage import SparseStorage
 from torch_sparse.tensor import SparseTensor
 
 
 def narrow(src: SparseTensor, dim: int, start: int,
-           length: int) -> SparseTensor:
+           length: int, bidim: False,
+           n_active_nodes: None) -> SparseTensor:
+    r"""It resizes :obj:`src` SparseTensor along one
+    specified dimension, or two dimensions at the same time
+    if the SparseTensor has 2 dimensions. In the
```

Comment:
Can we name this to something like `narrow_block` and move to its own function? Maybe we can also just name this `resize`?

Reply:
Yes, exatly.